common/persistence/persistence-tests/executionManagerTest.go

```diff
@@ -90,6 +90,8 @@ func (s *ExecutionManagerSuite) TestCreateWorkflowExecutionBrandNew() {
 		WorkflowTypeName:     workflowType,
 		WorkflowTimeout:      workflowTimeout,
 		DecisionTimeoutValue: decisionTimeout,
+		State:                p.WorkflowStateRunning,
```

Comment:
can you make sure we have a test that passes all combinations of {state,closeStatus} to each of {createWorkflow, updateWorkflow} APIs and makes sure GetWorkflowExecution() returns the expected value ? In the past, we have had atleast two bugs due to the lack of such a test

Reply:
sure